perf/get_test.go

```diff
@@ -17,7 +19,7 @@ func TestGetOptionsValidate(t *testing.T) {
 			name: "InvalidCedarOpts",
 			opts: GetOptions{
 				TaskID:    "task",
-				Execution: 0,
+				Execution: &zero,
```

Comment:
nit: You can just use `utility.ToIntPtr(0)` to achieve this (and the below ones).